cumulus_library/studies/core/builder_medication.py

```diff
@@ -106,6 +107,7 @@ def prepare_queries(self, cursor: object, schema: str, *args, **kwargs) -> dict:
             or medication_datasources["by_external_ref"]
         ):
             self.queries.append(
+                # core_templates.get_core_template()
                 get_core_medication_query(medication_datasources, has_userselected)
```

Comment:
I need to remove this, and the entry from templates, and wire it up to the core specific template loader - i don't think this is blocking review, though